Shared/Samples/Add feature collection layer from table/AddFeatureCollectionLayerFromTableView.swift

```diff
@@ -0,0 +1,178 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddFeatureCollectionLayerFromTableView: View {
+    /// A map with an ocean basemap style.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISOceans)
+        map.initialViewpoint = Viewpoint(
+            latitude: 8.849289,
+            longitude: -79.497238,
+            scale: 1e6
+        )
+        return map
+    }()
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: map)
+            .task {
+                do {
+                    // Adds feature collection layer to the map.
+                    let pointsTable = try await pointsCollectionTable()
+                    let linesTable = try await linesCollectionTable()
+                    let polygonsTable = try await polygonsCollectionTable()
```

Comment:
As written, these execute sequentially but using `async let` they could execute in parallel which would be better.